### ISSUE TITLE
Allow muting Elasticsearch deprecation warnings

### ISSUE THREAD

**Issue by jalogisch** [Graylog2/graylog2-server#10163]
## Expected Behavior
As a user, I may want to mute Elasticsearch deprecation warnings, because I cannot fix them immediately and want to prevent flooding logs with repeated messages.

There should be a configuration option in the config file that mutes ES deprecation warnings.

## Current Behavior

```
2021-02-24T08:30:07.169-05:00 WARN  [RestClient] request [GET http://1.1.1.1:9200/_cluster/settings?master_timeout=30s&include_defaults=true] returned 4 warnings: [299 Elasticsearch-7.10.2-747e1cc71def077253878a59143c1f785afa92b9 "[gateway.recover_after_nodes] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version."],[299 Elasticsearch-7.10.2-747e1cc71def077253878a59143c1f785afa92b9 "[discovery.zen.minimum_master_nodes] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version."],[299 Elasticsearch-7.10.2-747e1cc71def077253878a59143c1f785afa92b9 "[node.data] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version."],[299 Elasticsearch-7.10.2-747e1cc71def077253878a59143c1f785afa92b9 "[node.master] setting was deprecated in Elasticsearch and will be removed in a future release! See the breaking changes documentation for the next major version."]
```

## Context
Having such drastical warnings on startup will create a support burden. 

## Your Environment

* Graylog Version: 4.0

**Comment by abid1998** [comment]
What needs to be done in this ? I am new contributor.

**Comment by dennisoelkers** [comment]
Thanks for your interest, @abid1998! We already have a PR fixing this issue. If you are interested in contributing, looking into other issues from https://github.com/Graylog2/graylog2-server/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22 is probably a good idea.

Looking forward to your contributions!

**Comment by abid1998** [comment]
Ahh got it. Yeah will look for other issues. Any you recommend?

**Comment by janheise** [comment]
Fixed with PR#10239
